fgpv-vpgf/fgpv-vpgf
# feat(filters): add filters functionnalities

## Description


This PR finish the filters implementation relates to #1418. Check this issue to see what it is suppose to do.

## Testing

Manual testing in Chrome, FF and Safari
 - Check if value persist in filters when we switch table
 - Check if value for global search persist when we switch table
 - When we change something on table filters or setting filters information is updated properly
- We can apply the search on map (filter flag is shown) for feature layer and dynamic service layer
- Apply on Map button doesn't appear if not this type and for any user added layer
- We can clear the filters (clear the filter on map and remove the flag)
- Work together with the filter by extent function
- When sort or filtering, a processing message appears at the bottom of the map

## Documentation

Inline

## Checklist


- [x] commits messages follow the guidelines
- [x] code passes unit tests


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/fgpv-vpgf/fgpv-vpgf/1605)
